[DOCS] Remove Outdated Snapshot Docs (#61684)
elastic/elasticsearch: docs/reference/snapshot-restore/monitor-snapshot-restore.asciidoc

[discrete]
[[get-snapshot-stop-snapshot]]
=== Stop snapshot and restore operations

> If a currently running snapshot was started by mistake, or takes unusually long, it can be stopped using the <<delete-snapshot-api,delete snapshot API>>.

Reviewer: ```suggestion
To stop a currently running snapshot that was started by mistake or is taking unusually long, use
the <<delete-snapshot-api,delete snapshot API>>.
```

Reviewer: Suggest reordering this sentence to make it more focused on the action. I realize that I missed that change in the previous PR 😆
